src/Makefile

```diff
@@ -59,10 +66,16 @@ copy-scala: #remove this awful duplication ASAP
 	mv ctfp-reader-scala.pdf ../out/$(GIT_VER)/category-theory-for-programmers-scala.pdf
 	mv ctfp-print-scala.pdf ../out/$(GIT_VER)/category-theory-for-programmers-scala-print.pdf
 
+copy-ocaml: #remove this awful duplication ASAP
```

Comment:
Hopefully, someone will manage to improve on this makefile mess. I learned Makefiles from `i_have_no_idea_what_im_doing.pdf` :) Just like LaTeX!